util/util.go

```diff
@@ -116,6 +116,10 @@ func WriteFile(fs billy.Basic, filename string, data []byte, perm fs.FileMode) (
 	if err == nil && n < len(data) {
 		err = io.ErrShortWrite
 	}
+	err = f.Sync()
```

Comment:
We should call Sync for implementations that support it:

```suggestion
   if sf, ok := f.(Syncer); ok {
	  err = f.Sync()
```